scripts/src/main/resources/scripts/command/eclipse

```diff
@@ -254,6 +258,42 @@ function doImportEclipse() {
   "${ECLIPSE_HOME}/eclipse" -nosplash -vm "${ECLIPSE_JAVA_HOME}/bin/java" -application org.eclipse.ant.core.antRunner -buildfile "${DEVON_IDE_HOME}/scripts/lib_script/import.xml" -data "${WORKSPACE_PATH}" -keyring ~/.eclipse/.keyring -vm "${ECLIPSE_JAVA_HOME}/bin/java" -vmargs -DdevonImportPath="${importPath}" -DdevonImportWorkingSet="${importWorkingSets}" ${ECLIPSE_VMARGS} 
 }
 
+# $1: URL
+# $2: name
+# $3: optional dest
+function doMirrorUpdateSite() {
+  if [ -z "${2}" ]
+  then
+    doFail "Parameter <URL> or <name> is missing."
+  fi
+  local dest="${ECLIPSE_HOME}/update-sites/"
+  local timestamp
+  timestamp=$(date +'%m-%d-%Y')
+  local eclipsecli="${ECLIPSE_HOME}/eclipse"
+  if [ -n "${3}" ]
+  then
+    dest="${3}"
+  fi
+  if doIsWindows
+  then
+    eclipsecli="${ECLIPSE_HOME}/eclipsec.exe"
+  fi
+  "${eclipsecli}" -nosplash -verbose -application org.eclipse.equinox.p2.metadata.repository.mirrorApplication -source "${1}" -destination "${dest}/${2}_${timestamp}"
+  "${eclipsecli}" -nosplash -verbose -application org.eclipse.equinox.p2.artifact.repository.mirrorApplication -source "${1}" -destination "${dest}/${2}_${timestamp}"
```

Comment:
why is this line duplicated?

Reply:
There are two commands. The first one downloads the meta-data and the second one downloads the main content of the plugin.

Reply:
but there is absolutely no difference between the first and the second line. So your explanation does not make sense to me.

Reply:
OMH. Sorry. I thought I did look twice and more with care, but indeed you are correct that the one line contains `metadata` and the other one instead contains `artifact`. OK. Issue resolved.